ParseUI-Widget/build.gradle

```diff
@@ -39,7 +40,7 @@ android {
     buildToolsVersion "21.1.1"
 
     defaultConfig {
-        minSdkVersion 9
+        minSdkVersion 13
```

Comment:
@stanleyw @grantland  Thoughts on this? Right now for Parse, the minSdkVersion is 9. What is our standard to deprecate a version?